packages/terra-form-select/src/shared/_FrameUtil.js

```diff
@@ -25,31 +28,31 @@ class FrameUtil {
    * @param {ReactNode} target - The select wrapper.
    * @param {ReactNode} dropdown - The dropdown.
    * @param {number} maxHeight - The maxHeight of the dropdown dropdown.
+   * @param {Object} currentState - The current component state.
```

Comment:
This should be removed, since currentState is no longer used

Reply:
Removed fb1ae9e